### PR TITLE
Contacts modal

### PR DESCRIPTION
**Pull request by KageRyu7**
Created a modal to house links to all forms of contacts in Center and Hud.

### CODE REVIEW

File: app/js/components/contacts/ContactsWindow.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+var helpdeskPrefix = HELPDESK_ADDRESS.substring(0,7);
+
+var ContactInfo = React.createClass({
+
+    render: function () {
+        var feedbackTarget = this.isFeedbackEmail() ? "_self" : "_blank";
+        var helpdeskTarget = this.isHelpdeskEmail() ? "_self" : "_blank";
+        var requestTarget = "_self";
+        var socialTarget = "_self";
+        console.log (REQUEST_ADDRESS);
```
Comment (rpokorny) [inline app/js/components/contacts/ContactsWindow.jsx]: remove this

**Comment by wski** [comment]
Once rebased to one commit in correct format  `feat(Contact Modal): Added contact modal to global navbar` or similar it will be :+1:

**Comment by wski** [comment]
:+1:

**Comment by clarkjefcoat** [comment]
@KageRyu7: There should be spaces around Help Center

image

**Comment by clarkjefcoat** [comment]
@KageRyu7 There is an extra apostrophe here

image

**Comment by KageRyu7** [comment]
Fixed the spaces around help center and the double apostrophe. Anything else before I push?

**Comment by clarkjefcoat** [comment]
I didn't have anything else.

**Comment by KageRyu7** [comment]
changes pushed

**Comment by clarkjefcoat** [comment]
:+1:

**Pull request merged**